resources/core/charts/gateway/templates/kyma-gateway-certs.yaml

```diff
@@ -1,4 +1,5 @@
-{{ if not .Values.global.environment.gardener }}
+{{ if not .Values.global.newInstallation }}
```

Comment:
I don't like it. We were talking about that - we should not manage certs with helm. But introducing a global flag for that is ugly.

Reply:
This flag was introduced to support new and old way of managing the certificates in parallel. In the new way this secret is not created in this place, but in the certificates chart. Once we get rid of kyma-installer this secret can be removed, as well as xip-patch chart.

I set this flag to false as a default, so we don't have to modify the old way of installation in any way. Temporarily we need to set it to true when installing Kyma in the new way.

Reply:
Still I don't see any reason to introduce global flag. New installation should not install core, because you moved the secret and there is nothing more there. Old installation doesn't install certificates. Why do you need the flag?